### ISSUE TITLE
Should GOVERNANCE.md be required of all projects?

### ISSUE THREAD

**Issue by tobie** [openjs-foundation/project-onboarding#47]
The CPC cares about having open governance models for Foundation projects, yet only requires the governance model be documented for Impact-level projects in GOVERNANCE.md.

This sends a mixed-message.

Maybe extending that _documentation_ requirement to _every_ project would be a good thing. 
Even if a number of projects won't have open governance models at first, at least this creates transparency and documents the starting point.

**Comment by mhdawson** [comment]
I agree that we should all projects to document their governance model.

**Comment by bnb** [comment]
I agree with documenting governance model, though I'd request that it not be required to live in GOVERNANCE.md. Electron approaches this by having a centralized repository for its governance which feels like an equally valid approach.

**Comment by tobie** [comment]
Agreed. AMP, for example, only has a `GOVERNANCE.md` file in its `meta` repository. It doesn't live elsewhere either.

**Comment by jorydotcom** [comment]
All projects are required to document their decision structure somewhere (in a contributing.md file, on the readme, etc), only Impact projects are required to have a governance.md or Governance repo. we can clarify this on the template.

**Comment by tobie** [comment]
A little bit of consistency wouldn’t hurt, here. The governance model could be described in the charter, in `governance.md`, or in a dedicated repo, for example (referenced from the charter in the latter two examples).
